include/onnxruntime/core/graph/graph.h

```diff
@@ -679,6 +679,9 @@ class Graph {  // NOLINT(clang-analyzer-optin.performance.Padding): preserve exi
   /** Gets the path of the owning model, if any. */
   const Path& ModelPath() const;
 
+  /** Gets the path of the external initializer data, if any. */
+  const Path& ExternalIniPath() const;
```

Comment:
nit: 'ini' is pretty vague, especially when used deep in the code. would ExternalDataPath be clearer (and that's also the param name in the new API?